demisto_sdk/commands/secrets/secrets.py

```diff
@@ -277,8 +277,13 @@ def get_related_yml_contents(self, file_path):
         # if script or readme file, search for yml in order to retrieve temp white list
         yml_file_contents = ''
         # Validate if it is integration documentation file or supported file extension
-        if find_type(file_path) in [FileType.PYTHON_FILE, FileType.README, FileType.POWERSHELL_FILE]:
-            yml_file_contents = self.retrieve_related_yml(os.path.dirname(file_path))
+        try:
+            if find_type(file_path) in [FileType.PYTHON_FILE, FileType.README, FileType.POWERSHELL_FILE]:
+                yml_file_contents = self.retrieve_related_yml(os.path.dirname(file_path))
+        except ValueError as e:
+            if "has a structure issue of file type yml. Error was: while constructing a mapping" in str(e):
+                with io.open(file_path, mode="r", encoding="utf-8") as matching_yml_file:
+                    yml_file_contents = matching_yml_file.read()
```

Comment:
If you need the yaml as dict, please use [YAMLHandler](https://panw-global.slack.com/archives/G011E63JXPB/p1645438057926279) whenever possible. Consult with @ilaner or myself if needed.